code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -229,8 +231,12 @@
 
 /obj/item/lightreplacer/afterattack(atom/T, mob/U, proximity)
 	. = ..()
+
+	var/beam
```

Comment:
This var is useless, just check `!bluespacemode` in both if statements

Reply:
actually, it makes it so it doesnt play the beam effects if youre standing close to a lightbulb

Reply:
wait no it is useless i can check proximity twice